internal/storage/manager.go

```diff
@@ -99,7 +100,14 @@ func (manager *Manager) Store(ctx context.Context, ccid cid.Cid, filename string
 			return fmt.Errorf("cid is Undefined")
 		}
 		var ucid cid.Cid
-		ucid, err = manager.handleObjectUploadToIPFS(ctx, ccid, filename)
+		ucid, err = manager.IpfsStore.Upload(data)
+		if err != nil {
+			manager.ipfsFailureCount.Inc(1)
+			log.Errorf("ipfs store reported error: %v", err)
+
+			return err
+		}
+		manager.ipfsSuccessCount.Inc(1)
```

Comment:
How is this info being used later?

Reply:
can we feed it to grafana?

Reply:
yes...metrics export in prometheus format was added in bsp-agent. It can be enabled with `--metrics` and by default is served at 127.0.0.1:6061